plat/imx/imx8m/gpc_common.c

```diff
@@ -0,0 +1,231 @@
+#include <stdbool.h>
```

Comment:
Missing copyright header.

Also, the commit message should say "refactor", not "refact".

Reply:
sorry, my fault, I will fix it.